container/rkt/helpers.go

```diff
@@ -20,17 +20,56 @@ import (
 	"path"
 	"strings"
 
+	rktapi "github.com/coreos/rkt/api/v1alpha"
 	"github.com/golang/glog"
+	"golang.org/x/net/context"
 )
 
 type parsedName struct {
 	Pod       string
 	Container string
 }
 
-func verifyName(name string) (bool, error) {
-	_, err := parseName(name)
-	return err == nil, err
+func verifyPod(name string) (bool, bool, error) {
+	pod, err := cgroupToPod(name)
+
+	if err != nil || pod == nil {
```

Comment:
changed it, so that 0 pods isn't an error, just a nil, nil return, only propegate errors if an actual error state occurs